### DISCUSSION TITLE
Welcome to Kali-Linux Discussions!

### DISCUSSION THREAD

**Discussion by aw-junaid** [aw-junaid/Kali-Linux#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MikiBiH75000** [comment]
I ned help !!
I have installed Kali and in termux the white bar is not moving. As soon as I press sudo apt install, it asks me for my password and then I can't enter the password because the white bar is not responding. Can you help me? Thanks. Mike

**Reply by aw-junaid** [reply]
Send Images of the Error.

**Reply by aw-junaid** [reply]
And where did you install kali linux

**Reply by HariCipher** [reply]
It's will not respond just type your password and enter it will done

**Comment by aw-junaid** [comment]
Where did you install kali linux.

**Reply by HariCipher** [reply]
I installed inside virtual box from kali.org

**Comment by HariCipher** [comment]
I tried to use the phishing tool but it's showing unable to generate link.

**Reply by aw-junaid** [reply]
what error shows

**Reply by HariCipher** [reply]
![IMG_20250128_214623.jpg](https://github.com/user-attachments/assets/8fccbd3e-adbc-4cc2-a96f-77e37db800be)

It's showing this

**Reply by HariCipher** [reply]
Give me a solution

**Reply by aw-junaid** [reply]
It causing server side error: Try use local host option

**Reply by HariCipher** [reply]
It's still same error

**Reply by aw-junaid** [reply]
Message me on Discord and share your screen.
